geode-core/src/main/java/org/apache/geode/internal/offheap/MemoryAllocatorImpl.java

```diff
@@ -55,6 +57,16 @@
   public static final String FREE_OFF_HEAP_MEMORY_PROPERTY =
       GeodeGlossary.GEMFIRE_PREFIX + "free-off-heap-memory";
 
+  public static final int UPDATE_OFF_HEAP_STATS_FREQUENCY_MS =
+      SystemProperty.getProductIntegerProperty(
+          "update-off-heap-stats-frequency-ms").orElse(360000);
```

Comment:
I think "off-heap-stats-update-frequency-ms" reads a bit better. It helps connect "frequency" to "update"